packages/stream_video/lib/open_api/video/coordinator/model/api_error.dart

```diff
@@ -99,7 +99,7 @@ class APIError {
 
       return APIError(
         statusCode: mapValueOfType<int>(json, r'StatusCode')!,
```

Comment:
@deven98, should we add a table or guide here to indicate which error the number maps to?